tests/testServer/ServiceStreamingRpcs.cpp

```diff
@@ -103,3 +103,16 @@ ::grpc::Status ServiceStreamingRpcs::bidirectionalStreamNegateNumbers(
     }
     return grpc::Status();
 }
+
+::grpc::Status ServiceStreamingRpcs::bidirectionalStreamEchoNumberOrString(
+            ::grpc::ServerContext* context,
+            ::grpc::ServerReaderWriter< ::examples::NumberOrStringOneOf,
+            ::examples::NumberOrStringOneOf>* stream
+            )
+{
+    ::examples::NumberOrStringOneOf message;
+    while (stream->Read(&message)) {
+        stream->Write(message);
+    }
+    return grpc::Status();
```

Comment:
What Status would be returned here?

Reply:
Default constructed status is a "Good" status